src/aggregation/summable.c

```diff
@@ -0,0 +1,267 @@
+#include "postgres.h"
```

Comment:
the code here - for most part - has only been moved over from `count.c`. Only changes done should be about introducing the generic typing of contributions to accommodate both sum and count